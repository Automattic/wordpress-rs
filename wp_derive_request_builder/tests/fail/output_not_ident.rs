#[derive(wp_derive_request_builder::WpDerivedRequest)]
enum UsersRequest {
    #[contextual_get(url = "/users", params = &UserListParams, "output" = Vec<SparseUser>)]
    List,
}

fn main() {}
