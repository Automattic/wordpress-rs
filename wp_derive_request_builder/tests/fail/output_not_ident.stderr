error: Expecting key value pairs (url = "", params = FooParam, output = FooOutput)
 --> tests/fail/output_not_ident.rs:5:64
  |
5 |     #[contextual_get(url = "/users", params = &UserListParams, "output" = Vec<SparseUser>)]
  |                                                                ^^^^^^^^
